test/step/simpleStep.js

```diff
@@ -20,6 +20,9 @@ describe('A step', () => {
                 const ret = await st.run()
                 //then
                 assert.ok(ret.isOk)
+                assert.ok(ret.toString())
```

Comment:
The ToString and ToJson methods were not used in any test battery, I decided to add them in this simpler test, to test them and increase the coverage.

Reply:
(1) I think it is out of the scope of the test
(2) Since `toString` and `toJson` have calculated values it would be nice to have their own tests checking their output values

Reply:
Make sense! I'll create specific tests for increase coverage about tostring and tojson